CHANGELOG.rst

```diff
@@ -21,6 +21,23 @@ Removed
 Security
 ========
 
+[2022.1.2] - 2022-02-07
```

Comment:
Intentionally skipped version `2022.1.1`, assuming that it'll land on this PR #68